<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Ctl</class>
 <widget class="QFrame" name="Ctl">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>243</width>
    <height>115</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::NoFrame</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <widget class="QSlider" name="pwrSlider">
   <property name="geometry">
    <rect>
     <x>73</x>
     <y>25</y>
     <width>160</width>
     <height>19</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="toolTip">
    <string>AM transmit power</string>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
   <property name="singleStep">
    <number>5</number>
   </property>
   <property name="value">
    <number>50</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="invertedAppearance">
    <bool>false</bool>
   </property>
   <property name="invertedControls">
    <bool>false</bool>
   </property>
   <property name="tickPosition">
    <enum>QSlider::TicksBelow</enum>
   </property>
   <property name="tickInterval">
    <number>10</number>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>73</x>
     <y>5</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>0      Power out      100</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QProgressBar" name="MicProgressBar">
   <property name="geometry">
    <rect>
     <x>73</x>
     <y>90</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Mic audio display</string>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="QSlider" name="audioSlider">
   <property name="geometry">
    <rect>
     <x>73</x>
     <y>64</y>
     <width>160</width>
     <height>19</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Volume control</string>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
   <property name="value">
    <number>50</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="tickPosition">
    <enum>QSlider::TicksBelow</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>73</x>
     <y>47</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>1      Audio Gain     100</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>77</width>
     <height>116</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>-1</number>
    </property>
    <item>
     <widget class="QPushButton" name="btnMox">
      <property name="focusPolicy">
       <enum>Qt::ClickFocus</enum>
      </property>
      <property name="toolTip">
       <string>Toggle transmit mode</string>
      </property>
      <property name="text">
       <string>MOX</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnTune">
      <property name="toolTip">
       <string>Toggle tuning mode</string>
      </property>
      <property name="text">
       <string>Tune</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnMute">
      <property name="toolTip">
       <string>Mute/Unmute audio</string>
      </property>
      <property name="text">
       <string>Mute</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnMaster">
      <property name="toolTip">
       <string>Regain Master control</string>
      </property>
      <property name="text">
       <string>Master</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
